libs/common/src/lib/policies/financial-report.policy.ts

```diff
@@ -0,0 +1,10 @@
+import { UserPermissionsPolicy } from "./user-permissions.policy";
+import { FinancialReport } from "@terramatch-microservices/database/entities";
+
+export class FinancialReportPolicy extends UserPermissionsPolicy {
+  async addRules() {
+    if (this.userId != null) {
+      this.builder.can(["read", "delete"], FinancialReport);
+    }
```

Comment:
I think this needs a lot more protection - allowing any authenticated user to read and delete every financial report in the DB is very dangerous.